src/qibolab/instruments/qm/controller.py

```diff
@@ -127,6 +132,15 @@ class QMController(Controller):
     """Maximum bounds used for batching in sequence unrolling."""
     calibration_path: Optional[str] = None
     """Path to the JSON file that contains the mixer calibration."""
+    write_calibration: bool = False
+    """Require writing permissions on calibration DB."""
```

Comment:
Do we really need this? We could always just use the `tempfile` solution even when we have write access and it is not strictly required. I wouldn't expect a big overhead.

Reply:
My only concern is that the temporary file will always be erased upon disconnection. So you will always lose that.

Reply:
Sorry, I could not write this earlier. I just wanted to note that this is probably not relevant if we are just executing, as this shouldn't change the content of the file, but could be relevant if we perform the mixer calibration. Currently the driver provides some functions to do that, but I think they are not that useful (I even erased from 0.2). Still, it may indeed be useful to keep this option in case we do that in the future. Another point is that mixer calibration cannot be done without write access, so it will only be possible with the global cluster installation.

Reply:
Thanks @stavros11 for the comment.

I agree this option might be useless, but it also has a default, so it's no harm. For the time being, merging it was the conservative approach, but we can revert it later on.